lib/retriable.rb

```diff
@@ -0,0 +1,13 @@
+module Retriable
+  def retries(*rescue_errors)
+    retries ||= 0
```

Comment:
shadowing the method name is a little confusing - maybe call the variable `retry_count` instead?

Reply:
👍 good point, @willfish  thinks the same, lets go with `with_retries`